BE/LegalEntities/LEIEntities.rdf

```diff
@@ -367,14 +367,13 @@
 		<fibo-fnd-utl-av:adaptedFrom>GLEIF Level 2 Relationship Record (RR) Common Data Format (CDF), see https://www.gleif.org/en/about-lei/common-data-file-format/relationship-record-cdf-format#</fibo-fnd-utl-av:adaptedFrom>
 	</owl:NamedIndividual>
 	
-	<owl:ObjectProperty rdf:about="&fibo-be-le-lei;hasAddressOfLegalFormation">
+	<owl:ObjectProperty rdf:about="&fibo-be-le-lei;hasLegalAddress">
```

Comment:
This seems to be in the wrong ontology - legal address applies to many types of entity and the concept is not specific to those with a LEI. Best to have alongside hasHeadquartersAddress and hasRegisteredAddress.

Reply:
It was in this ontology originally, and the GLEIF does differentiate between legal and headquarters, which is, I think, how it ended up there in the first place.  There are many cases where the legal address is that of a third-party agent, though, which is one of the reasons I changed the name.  Also, many companies move, and don't retain the address where they were founded as their legal address.